index.html

```diff
@@ -948,10 +948,12 @@ <h3>
           <a>PerformanceResourceTiming</a> objects in the <a data-cite=
           "PERFORMANCE-TIMELINE-2#performance-timeline">Performance
           Timeline</a>. If the <a data-cite="FETCH#concept-tao-check">timing
-          allow check</a> algorithm fails for a resource, the following
-          attributes of its <a>PerformanceResourceTiming</a> object are set to
-          zero: {{PerformanceResourceTiming/redirectStart}},
+          allow check</a> algorithm fails for a resource, the corresponding
+          <a>PerformanceResourceTiming</a> object will have any attributes
+          exposing cross-origin data masked. Therfore, the following attributes
```

Comment:
Any data RE cross-origin resources is "cross-origin data". At the same time, the URL, start time and response end time are considered OK to expose as they are already exposed elsewhere. I tried to convey that with "unexposed cross-origin data", but apparently that's not clear enough. Any other wording that can capture that? "not-previously-exposed cross-origin data"?